packages/fx-core/src/component/generator/spfx/spfxGenerator.ts

```diff
@@ -708,21 +708,24 @@ export class SPFxGenerator {
         );
         for (let i = 1; i < webparts.length; i++) {
           const webpart = webparts[i];
-          const webpartManifestPath = path.join(
-            webpartsDir,
-            webpart,
-            `${webpart.split(path.sep).pop() as string}WebPart.manifest.json`
+          const webpartManifestFile = (await fs.readdir(path.join(webpartsDir, webpart))).find(
```

Comment:
Ubuntu file system is case sensitive, need to apply this change for updating Teams manifest file, too.